docs/DeveloperGuide.adoc

```diff
@@ -25,10 +25,14 @@
 . Rename the project if you want. Click `Next`
 . Ensure that your `\src` and `\test\java` folders are checked. Keep clicking `Next`
 . Click `Finish`
-. Add JUnit 4 to classpath
+. Add JUnit 5 to classpath
 .. Open any test file in `\test\java` and place your cursor over any `@Test` highlighted in red
-.. Press kbd:[ALT + ENTER] and select `Add 'JUnit4' to classpath`
-.. Select `junit:junit:4.12` and click `OK` (if `4.12` is not available, choose any `4.x` that is later than `4.12`)
+.. Press kbd:[ALT + ENTER] and select `Add 'JUnit5.3' to classpath` (if `5.3` is not available, any `5.x` will suffice)
++
+image::HelpfulIDEA.png[]
```

Comment:
Also, I wonder if a filename that describes the relevant steps in the documentation would be more helpful, e.g. (`AltEnterSelectJunit53.png`)